### DISCUSSION TITLE
BUG RSVP not displayed after last Update

### DISCUSSION THREAD

**Discussion by loelismurf** [liedekef/events-made-easy#324] [General]
after the Update the Placeholder do not fill out anything Example: https://www.pfadiriko.ch/events/uebertritt-21-10-2023

this is my code for a Singel Event (found under new Event -> Event Section Singel Event
```
#_STARTDATE<br />
#_TOWN<br />
#_NOTES<br />
Anmeldung<br />
#_ADDBOOKINGFORM<br />
<details>
  <summary>Abmeldung</summary>
  #_CANCELBOOKINGFORM
</details>
<br />
#_MAP
```

**Comment by loelismurf** [comment]
![image](https://github.com/liedekef/events-made-easy/assets/77551562/eb26287d-6ef8-4afc-942e-e6ec62d2749b)

**Reply by loelismurf** [reply]
![image](https://github.com/liedekef/events-made-easy/assets/77551562/3429132b-0cb7-4490-b593-2f14f2f38c59)
No HTML Code for the Placeholder generatet

**Reply by liedekef** [reply]
This is not an EME issue: you have javascript errors on your pages ("Uncaught ReferenceError: jQuery is not defined"), which is the cause of the problem. Fix those and all will work again.

**Reply by loelismurf** [reply]
I thank you very much for fast support.
Error was cause of the Update of Plugin "Ultimeter - the Ultimate Progress and Goals Meter" broke the Site
